opencog/cython/opencog/logger.pxd

```diff
@@ -0,0 +1,45 @@
+from libcpp cimport bool
+
+# basic wrapping for std::string conversion
+cdef extern from "<string>" namespace "std":
```

Comment:
I am not sure it is enough, but Cython has std::string wrapper:
```
from libcpp.string cimport string
```

Reply:
Thanks for your feedback @vsbogd, I tried to replace the cdef with `from libcpp cimport string` but I'm getting subsequent error messages like

```
        cLogger(string s)
               ^
------------------------------------------------------------

/home/nilg/OpenCog/atomspace/opencog/cython/opencog/logger.pxd:25:16: 'string' is not a type identifier
```

Since I'm not very familiar with Cython and this code was there before that change (it has just been moved around, not added) I'm gonna keep it that way for now. If you know how to completely fix it please provide more information though, any code simplification is always welcome.